tappsbt/interface.go

```diff
@@ -40,6 +40,7 @@ var (
 	PsbtKeyTypeInputTapAnchorTapscriptSibling             = []byte{0x78}
 	PsbtKeyTypeInputTapAsset                              = []byte{0x79}
 	PsbtKeyTypeInputTapAssetProof                         = []byte{0x7a}
+	PsbtKeyTypeInputTapSigHashType                        = []byte{0x7b}
```

Comment:
We don't need to encode this separately. Because the `i.SighashType` field is on the embedded `psbt.PInput`, it gets serialized automatically.